qtfred/README.md

```diff
@@ -10,26 +10,83 @@ a glimpse of hope appeared to users of other operating systems.
 
 **THIS TOOL IS STILL UNDER HEAVY DEVELOPMENT AND IS _NOT_ TO BE CONSIDERED STABLE.**
 
-Pre-requisites
---------------
+Build Instructions
+------------------
 
-  * CMake >= 2.8.12
-  * Qt5
+Additional Dependencies:
+
+- Qt >= 5.7
+
+qtFRED also needs all the depdencies that standard FSO needs.
```

Comment:
dependencies?

Reply:
A single word question isn't very useful. What do you want to say with this?

Reply:
I think it's just a typo; dependencies vs depdencies

Reply:
Oh, I didn't see that typo. Then this makes more sense.

Reply:
The typo should be fixed now.